hc5-async-json/src/test/java/com/ok2c/hc5/json/JsonNodeAssemblerTest.java

```diff
@@ -15,16 +15,14 @@
  */
 package com.ok2c.hc5.json;
 
-import java.io.InputStream;
```

Comment:
@schulzp Could import order be kept as it to reduce the noise in the change-set? Looks good otherwise.